mithril-aggregator/src/database/provider/certificate.rs

```diff
@@ -349,18 +338,9 @@ impl<'client> Provider<'client> for CertificateRecordProvider<'client> {
     }
 
     fn get_definition(&self, condition: &str) -> String {
-        let aliases = SourceAlias::new(&[("{:certificate:}", "c"), ("{:top_n:}", "top_n")]);
+        let aliases = SourceAlias::new(&[("{:certificate:}", "c")]);
         let projection = Self::Entity::get_projection().expand(aliases);
-
-        format!(
-            r#"with
-    top_n (max_rowid) as (select max(ROWID) from certificate)
-  select {projection}
-  from certificate as c 
-    cross join top_n
-  where {condition}
-  order by ROWID desc"#
-        )
+        format!("select {projection} from certificate as c where {condition} order by ROWID desc")
```

Comment:
Why does the changes from the soft limit reset commit are here ? You should rebase your PR on `main`.